.rubocop.yml

```diff
@@ -55,6 +82,9 @@ RSpec/ExampleLength:
 RSpec/MultipleExpectations:
   Enabled: false
 
+RSpec/MultipleMemoizedHelpers:
+  Enabled: false
```

Comment:
turning this off seemed to be in the same spirit as us consciously turning off `RSpec/MultipleExpectations`.  didn't seem like a useful rule, considering our current testing patterns.